UKHomeOffice/sgmr-service
# SPCR-482 Pass all pleasure craft data on the voyage PATCH

### AC / Description of changes
Update single PC component so that it includes the new PC fields on the voyage submission

### To Test

- make sure you have 1 PC on your account
- create a voyage
- continue to the PC page
- submit that page
- check the network tab to ensure AIS, MMSI, VAT, CharterOrHire questions are on the payload

### Notes

Due to a bug being addressed in SPCR-481, the submit from the PC page to the voyage WILL fail.

---

**Remember to merge to feature branches not `master`.**
